Find a functionally equivalent code:android.content.res.AssetManager.openXmlBlockAsset:COMMENT
Method Modifier: default     final       
Comment:/*package*/

@@ -1,14 +1,3 @@
 {
-    synchronized (this) {
-        if (!mOpen) {
-            throw new RuntimeException("Assetmanager has been closed");
-        }
-        int xmlBlock = openXmlAssetNative(cookie, fileName);
-        if (xmlBlock != 0) {
-            XmlBlock res = new XmlBlock(this, xmlBlock);
-            incRefsLocked(res.hashCode());
-            return res;
-        }
-    }
-    throw new FileNotFoundException("Asset XML file: " + fileName);
+    return openXmlBlockAsset(0, fileName);
 }

